src/stream/src/executor/global_simple_agg.rs

```diff
@@ -138,19 +137,7 @@ impl<S: StateStore> GlobalSimpleAggExecutor<S> {
     ) -> StreamExecutorResult<()> {
         let capacity = chunk.capacity();
         let (ops, columns, visibility) = chunk.into_inner();
-
-        // --- Retrieve all aggregation inputs in advance ---
-        let all_agg_input_arrays = agg_input_array_refs(agg_calls, &columns);
-        let pk_input_arrays = pk_input_array_refs(input_pk_indices, &columns);
```

Comment:
Can we also remove these two utility functions?

Reply:
yes we can, removed in the latest commit